microceph/common/network.go

```diff
@@ -0,0 +1,40 @@
+package common
+
+import (
+	"fmt"
+	"net"
+)
+
+// ValidateNetworkAddress validates the provided IP on host, returns the network
+// CIDR containing the provided address; empty string on failures.
+func ValidateNetworkAddress(address string) (string, error) {
```

Comment:
I feel like the name doesn't match the implementation. To me, `validate` alludes to essentially running some checks only; this function rather returns a network address instead (which doesn't seem to be used anywhere either?)

Reply:
done